xchange-core/src/main/java/org/knowm/xchange/dto/marketdata/OrderBook.java

```diff
@@ -183,54 +195,6 @@ private void updateDate(Date updateDate) {
     }
   }
 
-  @Override
-  public int hashCode() {
-
-    int hash = 17;
-    hash = 31 * hash + (this.timeStamp != null ? this.timeStamp.hashCode() : 0);
-    for (LimitOrder order : this.bids) {
-      hash = 31 * hash + order.hashCode();
-    }
-    for (LimitOrder order : this.asks) {
-      hash = 31 * hash + order.hashCode();
-    }
-    return hash;
-  }
-
-  @Override
-  public boolean equals(Object obj) {
-
-    if (obj == null) {
-      return false;
-    }
-    if (getClass() != obj.getClass()) {
-      return false;
-    }
-    final OrderBook other = (OrderBook) obj;
-    if (this.timeStamp == null
-        ? other.timeStamp != null
-        : !this.timeStamp.equals(other.timeStamp)) {
-      return false;
-    }
-    if (this.bids.size() != other.bids.size()) {
-      return false;
-    }
-    for (int index = 0; index < this.bids.size(); index++) {
-      if (!this.bids.get(index).equals(other.bids.get(index))) {
-        return false;
-      }
-    }
-    if (this.asks.size() != other.asks.size()) {
-      return false;
-    }
-    for (int index = 0; index < this.asks.size(); index++) {
-      if (!this.asks.get(index).equals(other.asks.get(index))) {
-        return false;
-      }
-    }
-    return true;
-  }
-
```

Comment:
why is this being removed?

Reply:
because the if/else clause already covers this case.

Reply:
Fixed